Simplenote/NSAttributedStringToMarkdownConverter.swift

```diff
@@ -0,0 +1,33 @@
+import Cocoa
+
+
+// MARK: - NSAttributedString to Markdown Converter
+//
+class NSAttributedStringToMarkdownConverter: NSObject {
+
+    /// Markdown replacement for "Unchecked Checklist"
+    ///
+    private static let unchecked = "- [ ]"
+
+    /// Markdown replacement for "Checked Checklist"
+    ///
+    private static let checked = "- [x]"
```

Comment:
I want to say these were localized in the iOS app. Any issue with not localizing the `x` here?

Reply:
Noting for the record: Chatted with Eric few days ago, the localized snippet is the input converter (which performs a case insensitive check).

Okay to leave it this way for the output converter, since nothing (should) break.

**Thank you!!!**